beetmoverscript/utils.py

```diff
@@ -175,17 +175,21 @@ def generate_beetmover_template_args(context):
 def _generate_beetmover_template_args_maven(task, release_props):
     tmpl_args = {
         'artifact_id': task['payload']['artifact_id'],
-        'branch': release_props['branch'],
-        'buildid': release_props['buildid'],
+        'branch': release_props.get('branch'),
+        'buildid': release_props.get('buildid'),
         'product': release_props['appName'],
```

Comment:
@JohanLorenzo Given the above statement, should we trim the product as well from `tmpl_arguments`?

Reply:
Done!